torch/utils/tensorboard/x2num.py

```diff
@@ -0,0 +1,34 @@
+# DO NOT alter/distruct/free input object !
```

Comment:
What does x2num mean? It seems like this nomenclature is taken from tensorboard? A module-level explanation would be helpful

Reply:
convert x into numpy, where x is framework's name. Renamed to contert_np.py